supertokens_python/recipe/thirdpartyemailpassword/email_delivery/services/backward_compatibility/__init__.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Dict, Union, cast
+
+from supertokens_python.ingredients.emaildelivery.types import \
+    EmailDeliveryInterface
+from supertokens_python.recipe.emailpassword.emaildelivery.service.backward_compatibility import \
+    BackwardCompatibilityService as EmailPasswordBackwardCompatibilityService
+from supertokens_python.recipe.emailpassword.interfaces import \
+    TypeEmailPasswordEmailDeliveryInput
+from supertokens_python.recipe.emailpassword.recipe_implementation import \
+    RecipeImplementation as EPRecipeImplementation
+from supertokens_python.recipe.emailpassword.types import User as EPUser
+from supertokens_python.recipe.emailpassword.utils import \
+    InputEmailVerificationConfig as EPInputEmailVerificationConfig
+from supertokens_python.recipe.emailpassword.utils import \
+    InputResetPasswordUsingTokenFeature
+from supertokens_python.recipe.emailverification.emaildelivery.service.backward_compatibility import \
+    BackwardCompatibilityService as \
+    EmailVerificationBackwardCompatibilityService
+from supertokens_python.recipe.emailverification.interfaces import \
+    TypeEmailVerificationEmailDeliveryInput
+from supertokens_python.recipe.emailverification.types import User as EVUser
+from supertokens_python.recipe.thirdpartyemailpassword.recipeimplementation.implementation import \
+    RecipeImplementation
+from supertokens_python.recipe.thirdpartyemailpassword.types import (
+    TypeThirdPartyEmailPasswordEmailDeliveryInput, User)
+from supertokens_python.supertokens import AppInfo
+
+if TYPE_CHECKING:
+    from supertokens_python.recipe.thirdpartyemailpassword.utils import \
+        InputEmailVerificationConfig
+
+
+class BackwardCompatibilityService(EmailDeliveryInterface[TypeThirdPartyEmailPasswordEmailDeliveryInput]):
+    app_info: AppInfo
+    ev_backward_comp_service: EmailVerificationBackwardCompatibilityService
+    ep_backward_comp_service: EmailPasswordBackwardCompatibilityService
```

Comment:
You only need `ep_backward_comp_service`. See node SDK

Reply:
Fixed.